versions/3.0.3.md

```diff
@@ -1968,20 +1956,6 @@ requestBody:
           externalValue: 'http://foo.bar/examples/address-example.txt'
 ```
 
-In a parameter:
-
-```yaml
-parameters:
-  - name: 'zipCode'
-    in: 'query'
-    schema:
-      type: 'string'
-      format: 'zip-code'
-      examples:
```

Comment:
Rather than deleting this, it should be moved up to be under parameter

Reply:
Done!